roles/reboot/tasks/main.yml

```diff
@@ -1,2 +1,6 @@
 ---
+# provide an empty iterator when a list is not provided
+# http://docs.ansible.com/ansible/playbooks_conditionals.html#loops-and-conditionals
 - include: ../../../common/ans_reboot.yaml
+  vars:
+    services: "{{ wait_for_services|default([]) }}"
```

Comment:
TBH,  I was concerned about a naming collision, but I will fix it to use the same everywhere.